docs/001_develop/03_client-capabilities/011_criteria/index.mdx

```diff
@@ -1,11 +1,42 @@
 ---
-title: 'Criteria'
-sidebar_label: 'Criteria'
-id: client-criteria
-keywords: [crieria]
+title: 'Foundation Criteria'
+sidebar_label: 'Foundation Criteria'
+id: foundation-criteria
+keywords: [web, criteria, foundation criteria, frontend, ui]
 tags:
-- criteria
-sidebar_position: 11
+  - web
+  - criteria
+  - foundation criteria
+  - frontend
+  - ui
 ---
 
-Coming soon...
+# Genesis Foundation UI Criteria Utils
+
+
+## Installation
+
+To enable this module in your application, follow the steps below.
+
+1. Add `@genesislcap/foundation-criteria` as a dependency in your `package.json` file. Whenever you change the dependencies of your project, ensure you run the `$ npm run bootstrap` command again. You can find more information in the [package.json basics](../../../../../web/basics/package-json-basics/) page.
+
+```json
+{
+  ...
+  "dependencies": {
+    ...
+    "@genesislcap/foundation-criteria": "latest"
+    ...
+  },
+  ...
+}
+```
+
+## [API Docs](./docs/api/index.md)
+
+## License
+
+Note: this project provides front-end dependencies and uses licensed components listed in the next section; thus, licenses for those components are required during development. Contact [Genesis Global](https://genesis.global/contact-us/) for more details.
+
+### Licensed components
+Genesis low-code platform
```

Comment:
Can get rid of this
```suggestion
```

Reply:
Did you mean to mark this as resolved as it looks like you didn't remove it yet?

Reply:
This is being auto generated from the foundation ui module. moved it to docs/index.mdx and renamed overview to index.mdx